cmake/modules/RootMacros.cmake

```diff
@@ -1702,15 +1702,25 @@ endfunction()
 #----------------------------------------------------------------------------
 function(ROOT_ADD_PYUNITTESTS name)
   if(pyroot_experimental)
-    set(ROOT_ENV ROOTSYS=${ROOTSYS}
-        PATH=${ROOTSYS}/bin:$ENV{PATH}
-        LD_LIBRARY_PATH=${ROOTSYS}/lib:${ROOTSYS}/lib/${python_dir}:$ENV{LD_LIBRARY_PATH}
-        PYTHONPATH=${ROOTSYS}/lib:${ROOTSYS}/lib/${python_dir}:$ENV{PYTHONPATH})
+    if(MSVC)
```

Comment:
We already have a PR that removes the use of `${python_dir}`, which was a remnant: https://github.com/root-project/root/pull/5761